### PR TITLE
Nutanix:virt-who config API cases support for SCA enabled

### PR DESCRIPTION
**Pull request by yanpliu**
Cases : PASS
```
(robottelo_vv) [root@dell-per740-68-vm-04 robottelo]# pytest ./tests/foreman/virtwho/api/test_nutanix_sca.py --disable-pytest-warnings -q
......                                                                                                                                                                                                      [100%]
6 passed, 6 deselected, 30 warnings in 509.62s (0:08:29)

```

### CODE REVIEW

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: tests/foreman/virtwho/api/test_esx.py

File: tests/foreman/virtwho/api/test_nutanix_sca.py [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+        :id: 15e8c907-0e4c-40be-94e1-9734a85c2195
+
+        :expectedresults: hypervisor_id option can be updated.
+
+        :CaseLevel: Integration
+
+        :CaseImportance: Medium
+        """
+        values = ['uuid', 'hostname']
+        for value in values:
```
Comment (JacobCallahan) [inline tests/foreman/virtwho/api/test_nutanix_sca.py:111]:
```suggestion
        for value in ['uuid', 'hostname']:
```

File: tests/foreman/virtwho/cli/test_nutanix_sca.py [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+        :id: 565228cd-2124-41ed-89b7-84ec6ff77213
+
+        :expectedresults: hypervisor_id option can be updated.
+
+        :CaseLevel: Integration
+
+        :CaseImportance: Medium
+        """
+        values = ['uuid', 'hostname']
+        for value in values:
```
Comment (JacobCallahan) [inline tests/foreman/virtwho/cli/test_nutanix_sca.py:106]:
```suggestion
        for value in ['uuid', 'hostname']:
```

**Review by JacobCallahan — CHANGES_REQUESTED** [review]
Small changes

**Review by JacobCallahan — APPROVED** [review]

File: tests/foreman/virtwho/api/test_esx.py [hunk 3]
```diff
@@ -177,7 +177,7 @@ def test_positive_debug_option(self, default_org, form_data, virtwho_config, tar
 
         /foreman_virt_who_configure/api/v2/configs/:id"
 
-        :id: be395108-3944-4a04-bee4-6bac3fa03a19
+        :id: 24bf4c5b-cc7a-40ac-9079-e59c73984081
```
Comment (Gauravtalreja1) [inline tests/foreman/virtwho/api/test_esx.py:180]: @yanpliu Could you please revert this change for the test case ids? it seems irrelevant and resolve merge conflict

**Review by Gauravtalreja1 — CHANGES_REQUESTED** [review]

**Review by shubhamsg199 — APPROVED** [review]
ACK

**Pull request merged**